tools/logger/CMakeLists.txt

```diff
@@ -11,3 +11,5 @@ target_include_directories(sof-logger PRIVATE
 	"${SOF_ROOT_SOURCE_DIRECTORY}/src/include"
 	"${SOF_ROOT_SOURCE_DIRECTORY}"
 )
+
+install(TARGETS sof-logger DESTINATION bin)
```

Comment:
What's the prefix for destination ? I'm assuming it can be set somewhere.

Reply:
Similar to --prefix from autotools:
https://cmake.org/cmake/help/latest/variable/CMAKE_INSTALL_PREFIX.html